packages/compass-crud/src/constants/document-narrow-icon-breakpoint.ts

```diff
@@ -0,0 +1 @@
+export const DOCUMENT_NARROW_ICON_BREAKPOINT = '1264px'; // This is the width at which the icons switch to narrow mode in the Documents tab.
```

Comment:
nit: Might as use an actual doc comment here.

```suggestion
/**
 * The width at which the icons switch to narrow mode in the Documents tab
 */
export const DOCUMENT_NARROW_ICON_BREAKPOINT = '1264px';
```